lighthouse-core/audits/installable-manifest.js

```diff
@@ -210,10 +209,15 @@ class InstallableManifest extends Audit {
     const errorReasons = i18nErrors.map(reason => {
       return {reason};
     });
-    /** DevTools InstallabilityErrors does not emit an error unless there is a manifest, so include manifestValues's error */
-    if (manifestValues.isParseFailure) {
-      errorReasons.push({
-        reason: manifestValues.parseFailureReason});
+
+    // If InstallabilityErrors is empty, double check ManifestValues to make sure nothing was missed.
+    // InstallabilityErrors can be empty erroneously in our DevTools web tests.
```

Comment:
For some reason, `InstallabilityErrors` is empty in our DevTools web tests even though no manifest was fetched. Assuming this is a valid possibility, we can fall back to the old manifest failure reason.